### DISCUSSION TITLE
Custom SSL

### DISCUSSION THREAD

**Discussion by huhas12** [AzuraCast/AzuraCast#3686] [Q&A]
Hello just need your help, 

Tried to set up my certificates, following the guide https://www.azuracast.com/extending/modifying-docker.html#common-examples

replaced all the necessary by making an override file with the following :

**version: '2.2'
  
services:
  nginx_proxy:
    volumes:
      - /home/xenia_ath/radio_just4music_live.crt:/etc/nginx/certs/radio.just4music.live.crt:ro
      - /home/xenia_ath/radio_just4music_live.key:/etc/nginx/certs/radio.just4music.live.key:ro
  stations:
    volumes:
      - /home/xenia_ath/radio_just4music_live.crt:/etc/nginx/certs/radio.just4music.live.crt:ro
      - /home/xenia_ath/radio_just4music_live.key:/etc/nginx/certs/radio.just4music.live.key:ro**

but getting error message back. Docker installed in google VM

**root@radio:/var/azuracast# docker-compose up -d
azuracast_redis_1 is up-to-date
Starting azuracast_stations ... 
Starting azuracast_stations ... error
ERROR: for azuracast_stations  Cannot start service stations: OCI runtime create failed: container_linux.go:370: st
arting container process caused: process_linux.go:459: container init caused: rootfs_linux.go:59: mounting "/home/x
enia_ath/radio_just4music_live.crt" to rootfs at "/var/lib/docker/overlay2/38ddd2b70197ccc58b772ca6e7bd3cdf2ef4dd10
ed55b0dc505a25dc552fd250/merged/etc/nginx/certs/radio.just4music.live.crt" caused: open /var/lib/docker/overlay2/38
ddd2b70197ccc58b772ca6e7bd3cdf2ef4dd10ed55b0dc505a25dc552fd250/merged/etc/nginx/certs/radio.just4music.live.crt: re
ad-only file system: unknown
ERROR: for stations  Cannot start service stations: OCI runtime create failed: container_linux.go:370: starting con
tainer process caused: process_linux.go:459: container init caused: rootfs_linux.go:59: mounting "/home/xenia_ath/r
adio_just4music_live.crt" to rootfs at "/var/lib/docker/overlay2/38ddd2b70197ccc58b772ca6e7bd3cdf2ef4dd10ed55b0dc50
5a25dc552fd250/merged/etc/nginx/certs/radio.just4music.live.crt" caused: open /var/lib/docker/overlay2/38ddd2b70197
ccc58b772ca6e7bd3cdf2ef4dd10ed55b0dc505a25dc552fd250/merged/etc/nginx/certs/radio.just4music.live.crt: read-only fi
le system: unknown
ERROR: Encountered errors while bringing up the project.**

**Comment by Vaalyn** [comment]
The problem with `read-only file system: unknown` was fixed already but is not yet included in a stable release. You'll need to update to the rolling-release channel.

https://www.azuracast.com/administration/system/release-channels.html
